.github/workflows/vmtest.yml

```diff
@@ -21,6 +23,13 @@ jobs:
             echo "GIT_SHA=$GIT_SHA" >> $GITHUB_ENV
         - name: Install dependencies
           run: sudo apt -y install qemu-system-x86 curl
+        - name: Wait for build to succeed
```

Comment:
By existing pipeline, you mean the `Build` workflow? 

I fully agree that this current PR would not allow us to measure the duration of the workflow properly. I am not intended to merge this PR, but I want to prototype other possibilities to understand the features of GH actions and limitations for our use case

Reply:
If you mean adding back the `vmtest` workflow to `build`, is how it was in the beginning, and if the timing is recorded, it might be way easier to revert to that, but let's see what Kemal says :)

This is turning out to be way more complicated that it should 😢

Reply:
> If you mean adding back the `vmtest` workflow to `build`, is how it was in the beginning

Yes

Reply:
WIP of this in https://github.com/parca-dev/parca-agent/pull/1193. Will close this PR